cpp/src/io/shp/polygon_shapefile_reader.cpp

```diff
@@ -103,7 +126,7 @@ cudf::size_type read_layer(const OGRLayerH layer,
 
```

Comment:
Can't comment directly on line 125, but please eliminate the C-style cast there. If a cast is needed there, it should probably be a `static_cast`.